safe_cli/main.py

```diff
@@ -118,11 +122,39 @@ def loop(self):
                 pass
 
 
-def build_safe_cli():
+def get_safe_from_owner(
+    owner: ChecksumAddress, node_url: str
+) -> Optional[ChecksumAddress]:
+    """
+    Show a list of Safe to chose between them and return the selected one.
+    :param owner:
+    :param node_url:
+    :return: Safe address of a selected Safe
+    """
+    ethereum_client = EthereumClient(node_url)
+    safe_tx_service = TransactionServiceApi.from_ethereum_client(ethereum_client)
+    safes = safe_tx_service.get_safes_for_owner(owner)
+    if len(safes):
+        # Show safes
+        for option, safe in enumerate(safes):
+            print_formatted_text(HTML(f"{option} - <b>{safe}</b> "))
+        option = choose_option_question(
+            "Select the Safe to initialize the safe-cli", len(safes)
+        )
+        if option is not None:
+            return safes[option]
+
+    print_formatted_text(
+        HTML(f"<ansired>No safe was found for the specified owner {owner}</ansired>")
+    )
+    return None
+
+
+def build_safe_cli() -> Optional[SafeCli]:
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "safe_address",
-        help="Address of Safe to use",
+        "address",
+        help="The address of the Safe, or an owner address if --is-owner is specified.",
```

Comment:
I don't have a strong opinion here, but maybe `--get-safes-from-owner` could be more explicit? This flag makes sense now as we are implementing it on the future, but using an optional flag to modify a mandatory parameter does sound confusing. With a better name I think at least it would be more clear